lively.halos/morph.js

```diff
@@ -2008,7 +2008,7 @@ export default class Halo extends Morph {
     const t = this.target;
     const topBar = this.topBar;
 
-    if (!t.isText) return;
+    if (!t.isText && !t.editorModeName === 'richText') return;
```

Comment:
Could the logic of this method be then moved into the editor plugin instead?

Reply:
Do you mean this whole method? Because I do not see how we can come out with some kind of check here in the halo, but we could of course check for the plugin and then invoke a method there.

Reply:
Yes, the entire method basically, since all the code that follows after the check is only run in the presence of the rich text editor plugin, correct?